### PR TITLE
Fix axis autorange on doubleClick

### PR DESCRIPTION
**Pull request by etpinard**
... on graphs w/ `fixedrange:true` axes and w/o set `_rangeInitial`

fixes https://github.com/plotly/plotly.js/issues/3347 cc @plotly/plotly_js

### CODE REVIEW

File: src/plots/cartesian/dragbox.js [hunk 1]
```diff
@@ -703,13 +703,14 @@ function makeDragBox(gd, plotinfo, x, y, w, h, ns, ew) {
...
                 ax = axList[i];
 
-                if(!ax._rangeInitial) {
-                    attrs[ax._name + '.autorange'] = true;
-                }
-                else {
-                    rangeInitial = ax._rangeInitial;
-                    attrs[ax._name + '.range[0]'] = rangeInitial[0];
-                    attrs[ax._name + '.range[1]'] = rangeInitial[1];
+                if(!ax.fixedrange) {
```
Comment (archmoj) [inline src/plots/cartesian/dragbox.js]: I thought we may need to check for having zero in the first element of range array here.
Here is a [demo](https://codepen.io/MojtabaSamimi/pen/vvyqgJ?editors=0010) which fails using with `yaxis2.range=[0,1]`.

File: test/jasmine/tests/click_test.js [hunk 2]
```diff
@@ -900,6 +901,96 @@ describe('Test click interactions:', function() {
...
+                expect(gd._fullLayout.xaxis._rangeInitial).toBe(undefined);
+                expect(gd._fullLayout.yaxis._rangeInitial).toBe(undefined);
+                expect(gd._fullLayout.yaxis2).toBe(undefined);
+            })
+            .then(function() { return Plotly.react(gd, fig); })
+            .then(function() {
+                _assert('after react into fig', {
+                    xRng: [0, 4],
+                    yRng: [-1, 5],
+                    y2Rng: [-1, 1]
```
Comment (archmoj) [inline test/jasmine/tests/click_test.js]: We may also add the case of [0,1] here.

Comment (etpinard) [reply test/jasmine/tests/click_test.js]: What's special about the `[0,1]` case?

File: src/plots/cartesian/dragbox.js (hunk 1 above)
Comment (archmoj) [reply src/plots/cartesian/dragbox.js]: The demo is actually working with [0,1]. So need to worry there.

File: test/jasmine/tests/click_test.js (hunk 2 above)
Comment (archmoj) [reply test/jasmine/tests/click_test.js]: The test is perfect the way it is.

**Comment by archmoj** [comment]
A :dancer: from my side.
Thank you @etpinard for the fix.

**Pull request merged**